cmd/bio-rd/is-is.go

```diff
@@ -34,7 +34,13 @@ func configureProtocolsISIS(isis *config.ISIS) error {
 		}
 	}
 
+	configuredInterfaces := isisSrv.GetInterfaces()
 	for _, ifa := range isis.Interfaces {
+		if strSliceContains(configuredInterfaces, ifa.Name) {
+			log.Debugf("ISIS: Interface %q is already configured", ifa.Name)
```

Comment:
Is this method to be intended to be idempotent?

Reply:
That is the idea.